Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/fragments/CameraFragment.kt

```diff
@@ -126,23 +130,17 @@ class CameraFragment : Fragment() {
     private val animationTask: Runnable by lazy {
         Runnable {
             // Flash white animation
-            overlay.foreground = Color.argb(150, 255, 255, 255).toDrawable()
+            fragmentCameraBinding.overlay.foreground = Color.argb(150, 255, 255, 255).toDrawable()
             // Wait for ANIMATION_FAST_MILLIS
-            overlay.postDelayed({
+            fragmentCameraBinding.overlay.postDelayed({
                 // Remove white flash animation
-                overlay.foreground = null
+                fragmentCameraBinding.overlay.foreground = null
                 // Restart animation recursively
-                overlay.postDelayed(animationTask, CameraActivity.ANIMATION_FAST_MILLIS)
+                fragmentCameraBinding.overlay.postDelayed(animationTask, CameraActivity.ANIMATION_FAST_MILLIS)
             }, CameraActivity.ANIMATION_FAST_MILLIS)
         }
     }
 
```

Comment:
I see a number of postDelayed in this PR. If's more reliable to use ViewTreeObserver or the methods doOnPreDraw/doOnLayout from KTX. See https://developer.android.com/kotlin/ktx

Reply:
good point, add an entry to the https://github.com/android/camera-samples/issues/393 to track it: if we do not want to address it in the PR, we could come back with a separate Pull Request later. thank you!